goagen/gen_app/test_generator.go

```diff
@@ -59,9 +60,8 @@ func (g *Generator) generateResourceTest() error {
 		return nil
 	}
 	funcs := template.FuncMap{
-		"isSlice":       isSlice,
-		"initArguments": initArguments,
-		"tempvar":       tempvar,
+		"isSlice": isSlice,
+		"escape":  escape,
```

Comment:
You mean `Generator` has `escape` as a method and `reservedNames` as a field, right?

In that way, Generator will have all reserved names in the service and use them for all test helpers. `reservedNames` also affects the functions which actually does not have reserved names. Is it OK?